app/MMB/src/scenes/travelDocument/menuItem/PassengerMenuItem.js

```diff
@@ -0,0 +1,69 @@
+// @flow strict
+
+import * as React from 'react';
+import { View, Platform } from 'react-native';
+import { createFragmentContainer, graphql } from '@kiwicom/mobile-relay';
+import { Translation } from '@kiwicom/mobile-localization';
+import { StyleSheet, Color, Icon, Touchable } from '@kiwicom/mobile-shared';
+import idx from 'idx';
+
+import PassengerSubtitle from './PassengerSubtitle';
+import PassengerMenuRightContent from './PassengerMenuRightContent';
+import type { PassengerMenuItem as PassengerType } from './__generated__/PassengerMenuItem.graphql';
+
+type Props = {|
+  +data: PassengerType,
+|};
+
+class PassengerMenuItem extends React.Component<Props> {
+  onPress = () => console.warn('TODO');
+
+  render = () => {
+    const date = idx(this.props.data, _ => _.travelDocument.expiration) || null;
+    const idNumber =
+      idx(this.props.data, _ => _.travelDocument.idNumber) || null;
+
+    return (
+      <Touchable onPress={this.onPress} disabled={idNumber !== null}>
```

Comment:
I don't think it should be disabled; if you look on the current app, it is possible to access the screen of passengers with filled information.

Reply:
Right, but it is actually a bug, I will send you a link on slack